fields/types/boolean/BooleanColumn.js

```diff
@@ -0,0 +1,23 @@
+var React = require('react');
+
+var BooleanColumn = React.createClass({
+
+	render: function() {
+		var value = this.props.data.fields[this.props.col.path];
+		if (value === true) {
```

Comment:
not sure that value will always be a boolean, the truth check would be safer (and more consistent with the jade implementation)

i.e `if (value) {`